hexkit/providers/akafka/provider.py

```diff
@@ -69,6 +70,16 @@ class KafkaConfig(BaseSettings):
         example=["localhost:9092"],
         description="A list of connection strings to connect to Kafka bootstrap servers.",
     )
+    security_protocol: str = Field(
+        "PLAINTEXT",
```

Comment:
This should be a str literal `Literal["PLAINTEXT", "SSL"]` or a boolean if only these two values are supported. Alternatively, an enum.